assets/js/talks-presentations.js

```diff
@@ -0,0 +1,49 @@
+var modal = document.querySelectorAll('.modal-window');
```

Comment:
When you consider an issue like #628, do you think that this script might be reused? If so, maybe a more generic file name and set of required datapoints is in order?

I suppose a different modal might be used, though...

Reply:
yeah @lee, you are right we would need a different script. Though we could reuse some code from this file.